datastore/datastore.go

```diff
@@ -203,13 +205,26 @@ func newClient(scope string, kv string, addr string, config *store.Config, cache
 		config = &store.Config{}
 	}
 
-	addrs := strings.Split(addr, ",")
+	var addrs []string
+
+	if kv == string(store.BOLTDB) {
+		// Parse file path
+		addrs = strings.Split(addr, ",")
+	} else {
+		// Parse URI
+		parts := strings.SplitN(addr, "/", 2)
```

Comment:
this doesnt work if the configuration is something like `--cluster-store=etcd://x.x.x.x:y/<path1>,a.b.c.d:e/<path2>`.
should we support different path-urls for each of the different servers pointed to by the --cluster-store ?